textBasedAdventure.js

```diff
@@ -1,5 +1,124 @@
+        gramsToOz = arrGrams[0] * arrGrams[1];
+    }
+    }
+     console.log("Cool! You did it! It's 680.40 grams!");
+
+     thatsDone = readline.question("This is where we learn a little chemistry! Enter for SCIENCE!")
+
+     let meltingPoint;
+meltingPoint = readline.question("Now, let's melt! We have to get up to the perfect melting point in order to manipulate the cocoa butter crystals in the chocolate, which give chocolate its irrisistible texture and shine. Do you know what the best temperature to melt chocolate is? Truthy or Falsy: chocolate melts at 108 F? Type true or false!!");
+if (meltingPoint === "true") {        
+  console.log("You got it!!");
+} else {
+  console.log("Nope! Chocolate melts at a warm 108 F! Not too far from our own body temperature! Coincidence? I think not!")
+}
+
+let theTempest = readline.question("We've melted, it's time for the REAL fun part...")
+
+let temperTime;
+temperTime = readline.question("...TEMPERING! All we have to do is bring the cocoa butter crystals in perfect alignment by slowly reducing heat and constantly agitating the chocolate, WHILE adding just 1% SEED chocolate to help the crystals get into their best shape, creating beautifully smooth, rich, streak-free chocolate, and we do that by cooling the warm chocolate down to 88.7 F! Could you subtract 88.7 from 108?");
+let arrTemper = [108, 88.7];
+for (i = 0; i < arrTemper.length; i++) {
+  if (arrTemper[0] - arrTemper[1] === 19.3) {    
+    temperTime += arrTemper
+    temperTime = arrTemper[0] - arrTemper[1];   
+  }
+}
+console.log("Woohoo! You tempered chocolate! You're a chocolatier! And now, now we're ready to roll!");
+
+let timeToMold;
+timeToMold = readline.question("Pour your newly tempered chocolate into the pre-cooled ice-tray mold. There are 8 x 5 cavities in this mold! Multiply to get the total number of truffles we'll end up with!");
+
+let str = "Nice! 40 truffles!! Let's put it in the fridge to cool and get ready to pop out of the mold!"
+const multiply = (num1, num2, str) => {
+  if (num1 * num2 === 40) {
+  } 
+    return str
+}
+console.log(multiply(8, 5, str))
+
+let bangIt;
+bangIt = readline.question("Bang the mold on the counter 3 times in order to release the truffles! Ready-Set-GO! Enter 'bang' now!")
+const bangManyTimes = (num, callback) => {
+  for (let i = 0; i < num; i++) {
+    callback(num)
+  }
+  return callback;
+  }
+bangManyTimes(3, (callback) => console.log("BANG!"));
+
+readyOrNot = readline.question("HOORAY! You did it! You made truffles! Are you ready to reap the rewards??")
+if (readyOrNot === "yes") {
+  console.log("Hell yeahhhhh")
+} else {
+  console.log("Shucks, looks like I'll have to eat it all myself ;)")
+}
+
+let chocolateTime = readline.question("CUE THE FANFAIR! The moment of truth---It's chocolate time! Enter to fall in love <3");
+
+    let numInput;
+    numInput= 1 < 10
+       numInput = readline.question("How many pieces? Pick a number from 0-10");
+   switch (numInput) {
```

Comment:
When you have a switch statement with the same output for several cases, you don't need to repeat the work `case` you separate each option with a comma. 

Ex: 
`case '9', '8', '7', '6':`
    `console.log("Cool! You have enough for sharing--or not! Come back for more! Thanks for playing!" );`